src/styles/blocks/review.scss

```diff
@@ -0,0 +1,9 @@
+.review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+
+  &__count {
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here, but it's not defined in this file. Ensure that this variable is defined in a parent file or imported correctly to avoid compilation errors.